app/src/main/java/com/github/h3lp3rs/h3lp/view/mainpage/CprRateActivity.kt

```diff
@@ -68,6 +69,7 @@ class CprRateActivity : AppCompatActivity() {
 
     // Stops the CPR rate
     private fun stop() {
+        // Commented since animations cause problems on Cirrus' emulator
```

Comment:
Maybe activate here but shut down the test assertion? This way we don't forget it when doing the APK.